src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -28,3 +29,52 @@
 .row .edu-intro-spacing {
   margin-bottom: 7rem;
 }
+
+
+.search-select-school-fieldset {
+  .search-controls {
+    display: flex;
+    flex-wrap: nowrap;
+    input {
+      margin-right: 10px;
+    }
+
+    button {
+      max-width: 150px;
+    }
+  }
+
+  .radio-button {
+    input[type="radio"] {
+    }
+
+    span {
+      background-color: #f1f1f1;
```

Comment:
do we have sass variables we can use for some of these style values (color, size, etc)?